test/consistency.jl

```diff
@@ -413,7 +413,7 @@ function consistent_nlps(nlps; rtol=1.0e-8)
   # If there are inequalities, test the SlackModels of each of these models
   if nlps[1].meta.ncon > length(nlps[1].meta.jfix)
     slack_nlps = [SlackModel(nlp) for nlp in nlps]
-    consistent_functions(slack_nlps)
+    consistent_functions(slack_nlps, exclude=[cons])
```

Comment:
With a `SlackModel`, lower bounds and upper bounds of constraints become lower / upper bounds of slack variables. So if both models are different constraints bounds, I cannot compare `cons` functions.

With linear constraints and a `MathOptNLPModel`, `l <= Ax + b <= u` is internally stored as `l - b <= Ax <= u - b`.

Reply:
Can you change the example here in `NLPModels` to `l - b <= Ax <= u - b`?